### ISSUE TITLE
@predictions Error, Mapping Template

### ISSUE THREAD

**Issue by ykbryan** [aws-amplify/amplify-cli#4346]
**Describe the bug**
Unable to use @predictions and it returns errors

**Amplify CLI Version**
4.20.0

**To Reproduce**
Follow the codes and steps found in https://docs.amplify.aws/cli/graphql-transformer/directives#usage-5

OR

you can `git clone https://github.com/ykbryan/amplify-vuejs-file-upload-rekognition`, `amplify init` and `amplify push` to see the same errors. 

**Expected behavior**
Able to query @preductions and returns the result. 

**Screenshots**

<img width="1333" alt="Screenshot 2020-05-19 22 50 06" src="https://user-images.githubusercontent.com/1001146/82524745-4bdb8780-9b62-11ea-9cd7-1cc9caf3c85d.png">

**Desktop (please complete the following information):**
 - OS: Mac
 - Node Version. v13.14.0

**Additional context**

I have created [a PR to fix this issue](https://github.com/aws-amplify/amplify-cli/pull/4317)/
The issue can be found in the AppSync resolver. By removing the `toJson`, the variable will be parsed correctly without the double quotes. 

![Screenshot 2020-05-19 22 56 51](https://user-images.githubusercontent.com/1001146/82524827-788f9f00-9b62-11ea-8c14-0048d5711453.png)

**Comment by attilah** [comment]
@ykbryan thanks for reporting the issue, it seems that you're sending the label in quotes and that's causing the problem? Can you confirm that it is the case and if you remove the quotes then everything works as expected?

From our end we have to add an additional check if the input is in quotes and unquote if to resolve these situations.

**Comment by ykbryan** [comment]
> @ykbryan thanks for reporting the issue, it seems that you're sending the label in quotes and that's causing the problem? Can you confirm that it is the case and if you remove the quotes then everything works as expected?
> 
> From our end we have to add an additional check if the input is in quotes and unquote if to resolve these situations.

Yes, if you remove the quotes, everything works. Take a look at the earlier screenshot of the chrome console, the name of the s3 object has double quotes in the value.

I re-did another demo and I am following the exact codes in https://docs.amplify.aws/cli/graphql-transformer/directives#usage-5 This is built it in reactjs. The problem still exists. How do i remove the quotes from javascript? I just put in the "key" as variables. 
```
  const inputObj = {
    translateText: {
      sourceLanguage: "en", targetLanguage: "es" },
    identifyText: { key },
    convertTextToSpeech: { voiceID: "Conchita" }
  };
```

**Comment by edwardfoyle** [comment]
The fix for this has been released in CLI v 4.32.0
